File: Iniciante/1050/1050.java
import java.util.Scanner;

public class Main {
  public static void main(String[] args) {
    Scanner scanner = new Scanner(System.in);

    int ddd = Integer.parseInt(scanner.nextLine());

    String result = "";

    switch (ddd) {
      case 61:
        result = "Brasilia";
        break;
      case 71:
        result = "Salvador";
        break;
      case 11:
        result = "Sao Paulo";
        break;
      case 21:
        result = "Rio de Janeiro";
        break;
      case 32:
        result = "Juiz de Fora";
        break;
      case 19:
        result = "Campinas";
        break;
      case 27:
        result = "Vitoria";
        break;
      case 31:
        result = "Belo Horizonte";
        break;
      default:
        result = "DDD nao cadastrado";
    }

    System.out.println(result);
  }
}

File: Iniciante/1050/1050.rs
use std::io;

fn main() {
  let mut input = String::new();

  io::stdin().read_line(&mut input).unwrap();

  let ddd = input.trim().parse::<u16>().unwrap();

  let x = match ddd {
    61 => "Brasilia",
    71 => "Salvador",
    11 => "Sao Paulo",
    21 => "Rio de Janeiro",
    32 => "Juiz de Fora",
    19 => "Campinas",
    27 => "Vitoria",
    31 => "Belo Horizonte",
    _ => "DDD nao cadastrado",
  };

  println!("{}", x);
}